src/datetime/tests.rs

```diff
@@ -307,7 +307,7 @@ fn test_nanosecond_range() {
     // Far beyond range
     let maximum = "2262-04-11T23:47:16.854775804UTC";
     let parsed: DateTime<Utc> = maximum.parse().unwrap();
-    let beyond_max = parsed + TimeDelta::days(365);
+    let beyond_max = parsed + Days::new(365);
```

Comment:
Hmm, maybe we should resurrect your work on `CalendarDuration`?

Reply:
I'd like to. Part 1 is ready for review :smile:. Maybe wait a few more weeks until we are good underway with the `Result` stuff?